includes/class-wc-payments-payment-method-messaging-element.php

```diff
@@ -50,11 +50,8 @@ public function init(): string {
 		global $product;
 		$price           = $product->get_price();
 		$currency_code   = get_woocommerce_currency();
-		$billing_country = WC()->countries->get_base_country();
-
-		if ( WC()->customer ) {
-			$billing_country = WC()->customer->get_billing_country(); // Use the customer's billing country if available.
-		}
+		$base_country    = WC()->countries->get_base_country();
```

Comment:
Is that store country? If so, please rename the variable.

Reply:
Renamed in [b1fab5a](https://github.com/Automattic/woocommerce-payments/pull/6631/commits/b1fab5ad49878346aa3c5ff84f6f16293b93e9c6) thanks!